src/client/attestation/state.rs

```diff
@@ -35,13 +34,22 @@ pub enum State {
 
     /// Idle state with optional timeout. After timeout resets into Init state.
     Idle { timeout: Option<u64> },
+
+    /// Triggered when attestation is completed, either successfully or with
+    /// failure. This state is responsible for signalling attestation result
+    /// using LED.
+    Completion {
+        attestation_success: bool,
+        timeout: u64,
+    },
 }
 
 impl State {
     /// Transition into error state.
     pub fn error(&mut self) {
```

Comment:
Something similar should also be done for provisioning.

Reply:
Maybe we should also report success of provisioning by shorter/longer/blinking green LED?

Reply:
@krystian-hebel Done, when provisioning succeeds/fails Fobnails gives 3 quick blinks.
Also I reduced default retry limit from infinite to 5 (with timeout of 1 second) so that Fobnail signals failure after 5 of no response from attester.